src/components/themed/SelectFioAddress.js

```diff
@@ -81,6 +91,12 @@ class SelectFioAddressComponent extends React.PureComponent<Props, LocalState> {
 
   componentDidUpdate(prevProps: Props) {
     const { fioRequest, isSendUsingFioAddress } = this.props
+    const { bundlesUpdated } = this.state
+    if (bundlesUpdated) {
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name